### ISSUE TITLE
SSR friendly constants

### ISSUE THREAD

**Issue by xvaara** [cdmoro/bootstrap-vue-3#139]
Hey, just did some test with upcoming bootstrap 3.2 since it has fixed problems in their javascript with SSR node environment.

https://github.com/twbs/bootstrap/pull/34989

Noticed you're using HTMLElement as prop type in dropdown and it's not available on node environment so you need to override it: 

 ```
 export const HTMLElement = typeof window !== undefined
  ? window.HTMLElement
  : class HTMLElement extends Object {}
```

in bootstrapVue:
https://github.com/bootstrap-vue/bootstrap-vue/blob/1d59417df6869e2b04c651f6caeed9474cf14a84/src/constants/safe-types.js

btw. thanks for doing this. Gonna contribute something in the future.

**Comment by cdmoro** [comment]
resolved in #142

**Comment by xvaara** [comment]
You don't have to import them to type typescript. HTMLElement is type in typescript, not an object (that is missing from node).
